index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
The title used for the project doesn't really give a clear description of the project. I kindly suggest that you use a suitable name like "Test Suite FCC" or "My Trial Project" instead of "Fork Me! FCC: Test Suite Template". This will give the client a clear description of what the project is about.